test/ha/activator_test.go

```diff
@@ -38,8 +37,8 @@ import (
 const (
 	activatorDeploymentName = "activator"
 	activatorLabel          = "app=activator"
-	minProbes               = 400  // We want to send at least 400 requests.
-	SLO                     = 0.99 // We permit 0.01 of requests to fail due to killing the Activator.
+	minProbes               = 400 // We want to send at least 400 requests.
```

Comment:
We can lower this if we're checking 100% SLO

Reply:
We probably can, but I want to make sure we continue probing through the shutdowns and restarts.